src/commandHandlers/issue.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { load as cheerioLoad } from 'cheerio';
+import { Message, MessageEmbed } from 'discord.js';
+
+/**
+ * This command lets the user create an issue from discord to a github repository
```

Comment:
is an issue being created in this function? I think this command only returns an URL right?

Reply:
What I understood was the bot would return the URL so the can create it themselves. Is it supposed to create the issue by itself?

Reply:
oh sorry, what I understood from the comment is that an issue is created. If it returns an URL then it's fine 😅. Maybe we can change that comment to make it more explicit that the issue isn't created